gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -59,12 +64,15 @@
   private final Subject<Boolean> locationLockChangeRequests;
   private final Subject<CameraUpdate> cameraUpdateSubject;
   private final MutableLiveData<Event<Nil>> showMapTypeSelectorRequests = new MutableLiveData<>();
+  private final LiveData<ImmutableSet<String>> mbtiles;
+  private final List<MapBoxOfflineTileProvider> tileProviders = new ArrayList<>();
```

Comment:
Could you kindly add a TODO and Code health task to create our own wrapper interface/impl for these?

Reply:
f9a41a2650397ca3f118ba6677bb57e2affb43af